src/content/community/grants/merge-data-challenge/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Merge data challenge
+description: Instructions, deadlines, and prize information for the Merge data challenge
+lang: en
+sidebar: true
+sidebarDepth: 2
+---
+
+# The Merge data challenge {#the-merge-data-challenge}
+
+_Up until four weeks after the Merge, the Ethereum Foundation is sponsoring a Merge data analysis and data visualization blog post challenge. Here are all the details you need._
+
+<Divider />
+
+![Two people holding hands in field with ethereum logo](./merge-data-challenge.jpg)
```

Comment:
This is decorative, so it should have an empty alt.

```suggestion
![](./merge-data-challenge.jpg)
```

Reply:
Sorry I missed this. I don't, but tomo might